.travis.yml

```diff
@@ -24,6 +25,11 @@ cache:
   - third_party/v8/buildtools/linux64/
   - third_party/v8/third_party/llvm-build/
 
+before_cache:
```

Comment:
The before_cache steps just prior to saving the cache.
So to retain our sanity, maybe move this section to the end of the file?

Reply:
done